docs/_posts/2015-07-23-new-react-devtools-beta.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "New React Devtools Beta"
+author: jaredly
+---
+
+We've made an entirely new version of the devtools, and we want you to try it
+out!
+
+![The full devtools gif](/react/img/blog/devtools-full.gif)
+
+## Why entirely new?
+
+Perhaps the biggest reason was to create a defined API for dealing with
+internals, so that other tools could benefit as well and not have to depend on
+implementation details. This gives us more freedom to refactor things
+internally without worrying about breaking tooling.
+
+The current version of the devtools is a fork of Blink's "Elements" pane, and
+is imperative, mutation-driven, and tightly integrated with Chrome-specific
+APIs. The new devtools are much less coupled to Chrome, and easier to reason
+about thanks to React.
+
+## What are the benefits?
+
+- 100% React
+- firefox compatible
+- React-Native compatible
+- more extensible &amp; hackable
+
+## Are there any new features?
+
+Yeah!
+
+### The Tree View
+
+![The new tree view of the devtools](/react/img/blog/devtools-tree-view.png)
+
+- Much richer view of your props, including the contents of objects and arrays
+- Custom components are emphasized, native components are de-emphasized
+- Stateful components have a red collapser
+- Improved keyboard navigation (hjkl or arrow keys)
+- Selected component is available in the console as `$r`
+- Props that change highlight in green
+- Right-click menu
+
+  - Scroll node into view
+  - Show the source for a component in the "Sources" pane
+  - Show the element in the "Elements" pane
+
+### Searching
+
+Select the search bar (or press "/"), and start searching for a component by
+name.
+
+![](/react/img/blog/devtools-search.gif)
+
+### The Side Pane
+
+- Now shows the `context` for a component
+- Right-click to store a prop/state value as a global variable
+
+![](/react/img/blog/devtools-side-pane.gif)
+
+## How do I install it?
+
+For the beta, you need to [download the .crx](https://github.com/facebook/react-devtools/releases)
+and drag it into your `chrome://extensions` page. You will also need to
+disable the version you got from the Chrome web store.
```

Comment:
nit: move "Disable Chrome web store version" to the beginning of the paragraph, otherwise it's very easy to miss.